File: Q14/main.cpp
#include<iostream>
using namespace std;

int main(){
    int row,col;
    cin>>row>>col;

    int arr[row][col];

    for(int i=0;i<row;i++){
        for(int j=0;j<col;j++){
            cin>>arr[i][j];
        }
    }

    int minsum = 0;
    int minrow = 0;

    for(int i=0;i<row;i++){
        int sum = 0;
        for(int j=0;j<col;j++){
            sum += arr[i][j];
        }

        if(sum<minsum){
            minsum = sum;
            minrow = i;
        }
    }

    for(int j=0;j<col;j++){
        cout<<arr[minrow][j]<<" ";
    }
    return 0;
}

File: Q14/main.py
row = int(input())
col = int(input())

arr = []

for i in range(row):
    current_row = list(map(int, input().split()))
    arr.append(current_row)

min_sum = float('inf')
min_row = 0

for i in range(row):
    current_sum = sum(arr[i])

    if current_sum < min_sum:
        min_sum = current_sum
        min_row = i

for j in range(col):
    print(arr[min_row][j], end=" ")